net-p2p/eiskaltdcpp/eiskaltdcpp-2.4.1.ebuild

```diff
@@ -0,0 +1,184 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+LUA_COMPAT=( lua5-1 lua5-2 )
+
+PLOCALES="be bg cs de el en es eu fr hu it pl pt_BR ru sk sr sr@latin sv_SE tr uk vi zh_CN"
+
+inherit cmake l10n lua-single xdg-utils toolchain-funcs
+[[ ${PV} = *9999* ]] && inherit git-r3
+
+DESCRIPTION="Qt/DC++ based client for DirectConnect and ADC protocols"
+HOMEPAGE="https://github.com/eiskaltdcpp/eiskaltdcpp"
+
+LICENSE="GPL-2 GPL-3"
+SLOT="0"
+IUSE="cli daemon dbus +dht examples -gold -gtk idn -javascript +json libcanberra libnotify lua +minimal pcre +qt5 spell sqlite upnp -xmlrpc"
```

Comment:
These `-` don't do much and aren't used today.

Reply:
Removed them.